_docs/getting-started/mac.md

```diff
@@ -265,6 +242,14 @@ malloc error | If you are using homebrew leveldb on a Mac, you might see an erro
 
 <block class="mac prebuilt" />
 
+### Anaconda package
+
+```bash
+conda install -c caffe2 caffe2
```

Comment:
Nice! Once we're more comfortable with this let's make it the default install path. Do you know if Sierra and High Sierra both work with this?